source/educators/how-tos/data/manage_course_grades.rst

```diff
@@ -251,7 +251,7 @@ The gradebook includes the following features:
   specific graded subsection. For information about how to do this, see
   :ref:`Override in Gradebook`.
 
-* **Master’s courses:** The edX gradebook has several features for high-touch courses. For learners in the Master’s track:
+* **Master’s courses:** The Open edX gradebook has several features for high-touch courses. For learners in the Master’s track:
```

Comment:
I think we can remove everything about master's courses as that's edx.org-specific, but I'd check in with @jswope00 about that.

Reply:
@jesgreco please remove all content about Master's courses

Reply:
agree that references to "Master's courses" should be removed.